packages/create-codes/src/index.ts

```diff
@@ -332,6 +205,61 @@ async function run() {
     spaces: 2,
     EOL: os.EOL,
   });
+}
+
+/**
+ * Remove ESLint-related config from output project files.
+ */
+function removeEslintConfig() {
+  // TODO: copy .eslintignore from base
+  // fse.removeSync(path.resolve(appDir, ".eslintrc.js"));
+  //@ts-expect-error
+  delete packageObjs.scripts.lint;
+  //@ts-expect-error
+  Object.keys(packageObjs.devDependencies).forEach((key) => {
+    //@ts-expect-error
+    eslintPackages.includes(key) && delete packageObjs.devDependencies[key];
+  });
+}
+
+async function run() {
+  const { input } = meow(help, {
+    flags: {
+      help: { type: "boolean", default: false, alias: "h" },
+      version: { type: "boolean", default: false, alias: "v" },
+    },
+  });
+
+  const [dir] = input;
+
+  console.log("\nCreate Codes\n");
+  console.log("Welcome!\n");
+
+  const appDir = path.resolve(process.cwd(), dir ? dir : await promptAppDir());
+  const sharedConfigDir = path.resolve(CONFIG_TEMPLATES, "__shared");
+  const { jsLibrary, apiSolution, modules, tests } = await promptUserInput();
+
+  const templateName = jsLibrary;
+  await cloneTemplateToTempDir(templateName);
+
+  // Copy base codes
+  copyBase(appDir, tests?.useEslint ?? false);
+
+  // Copy modules
+  copyModules(appDir, apiSolution, tests !== null);
+
+  // Copy testing libraries
+  if (tests !== null) {
+    await copyTests(tests, jsLibrary, appDir, sharedConfigDir);
+  }
+
+  // FIXME: temporary processing, It would be good to refer to it from package.json under cli templates as well as others.
+  if (!tests?.useEslint) {
```

Comment:
ESLint is separated from `copyTests` because of how it differs from the other test modules; rather than adding to the output project, it removes related packages and script from the `package.json`.

(I feel like there's a better way to handle this...)